specs/create_password/01_main.rst

```diff
@@ -0,0 +1,167 @@
+==========================================================
+Create password
+==========================================================
+
+=====  ===============  =========  ==========
+epic        title        author     created
+=====  ===============  =========  ==========
+`65`_  Create password  tschubotz  2019-01-22
+=====  ===============  =========  ==========
+
+.. _65: https://github.com/gnosis/safe/issues/65
+
+.. _Main:
+
+
+#. `Main`_
+#. `User Interface`_
+
+.. sectnum::
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+Problem Definition
+---------------------
+
+- We need the user to set a password for the app so:
+
+  - Nobody can open the app or confirm transactions without the password
+  - We can encrypt the private key of the device.
+
+- Most devices have a fingerprint sensor. Users should be able to use it
+  additionally to the password.
+- We need the user to agree to the Terms and Privacy Policy.
+
+Inputs
+-----------
+
+- New password
+
+  - Minimum 8 characters (>=8)
+  - At least 1 number
+  - At least 1 letter
+  - No more than 2 identical characters in a row
+    (e.g. 111 is not allowed, 11 is fine)
+  - Maximum length?
```

Comment:
any updates on this? (personally I would be against maximum length)

Reply:
@rmeissner why would you be against the maximum length? Don't you think that adds some security for the user inputs?